### PR TITLE
Adjust the places/link icon to be 16x16 and of color black 

### PR DESCRIPTION
**Pull request by te-online**
This adjusts the places/link icon to the style of the other icons.
E.g. for use in menus.

Before:
bildschirmfoto 2017-08-29 um 11 22 39
bildschirmfoto 2017-08-29 um 11 22 27

After:
bildschirmfoto 2017-08-29 um 11 23 11
bildschirmfoto 2017-08-29 um 11 23 20

### CODE REVIEW

File: core/img/places/link.svg [hunk 1]
```diff
@@ -1 +1 @@
...
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16"><path d="M7.4 0C3.3 0 0 3.3 0 7.4s3.3 7.4 7.4 7.4 7.4-3.3 7.4-7.4S11.5 0 7.4 0zm.8.9c1.3 0 2.4.8 3.5 1.3l1.8 2.5-.3 1.1.6.3V8.5c-.2.7-.6 1.3-.9 2-.2.1 0-.8-.1-1 0-.6-.5-.6-.9-.2-.4.3-1.4.3-1.5-.4-.3-.8 0-1.7.3-2.5l-.6-.7.2-1.8-.8-.9.2-1-1-.6c-.2-.2-.6-.2-.7-.4.1 0 .2-.1.2-.1zM5.6 1s.1 0 .1.1c.4.2-.1.4-.2.6-.5.3.3.7.5 1 .4-.1.8-.7 1.4-.5.7-.2.6.6 1.1 1 .1.2.9.8.4.6-.5-.4-1-.4-1.3.1-.8.5-.3-.9-.7-1.2-.6-.7-.4.5-.4.9-.4 0-1.1-.3-1.5.2l.4.6.5-.7c0-.3.1.2.3.3.1.2.8.7.3.9-.8.4-1.4 1.1-2.1 1.7-.2.5-.7.4-1 0-.7-.4-.7.7-.6 1.1l.6-.4v1.1c-.4.4-.9-.7-1.3-.9V5.9c0-.4-.1-.9 0-1.3.8-.9 1.7-1.9 2.2-3h.8c.6.2.3-.7.5-.6zM4.4 9.2c.1 0 .2 0 .3.1.8.1 1.4.7 2 1.1.5.5 1.6.3 1.7 1.2-.2.9-1.1 1.4-1.8 1.7-.2.1-.4.2-.6.2-.7.2-1-.6-1.2-1.1-.3-.7-1.1-1.2-1-2.1 0-.4.2-1 .6-1.1z"/></svg>
```
Comment (MorrisJobke) [inline core/img/places/link.svg]: Could you add the view box: `viewbox="0 0 16 16"` because last time firefox otherwise didn't scaled properly.

Comment (te-online) [reply core/img/places/link.svg]: Sure!

**Comment by jancborchardt** [comment]
@te-online ideally squash the two commits into one, then we can merge it. :)

**Comment by te-online** [comment]
@jancborchardt I have included this in #6297.

**Pull request closed without merging**